bitnami/fluentd/README.md

```diff
@@ -84,6 +84,7 @@ The following tables lists the configurable parameters of the fluentd chart and
 | `forwarder.service.loadBalancerSourceRanges`    | Addresses that are allowed when service is LoadBalancer                                                        | `[]`                                                                                                    |
 | `forwarder.service.clusterIP`                   | Static clusterIP or None for headless services                                                                 | `nil`                                                                                                   |
 | `forwarder.service.annotations`                 | Annotations for the forwarder service                                                                          | `{}`                                                                                                    |
+| `forwarder.persistence.enabled`                 | Enable persistence volume for the forwarder                                                                          | `false`                                                                                                    |
```

Comment:
I'm missing `forwarder.persistence.hostPath.path` parameter here

Reply:
You're right, added. Thanks.